Tests/NIOOpenSSLTests/OpenSSLIntegrationTest.swift

```diff
@@ -754,10 +766,15 @@ class OpenSSLIntegrationTest: XCTestCase {
         // synchronization: all of this should succeed synchronously. If it doesn't, that's
         // a bug too!
         let closePromise = serverChannel.close()
-        XCTAssertFalse(closePromise.fulfilled)
+        closePromise.whenComplete {
```

Comment:
why this instead of `XCTAssertFalse(closePromise.isFulfilled)`?

Reply:
because isFulFilled is internal ?

Reply:
aaaah :)